db/pre-re/item_db_equip.yml

```diff
@@ -25687,19 +25687,15 @@ Body:
     Script: |
       bonus bStr,2;
   - Id: 5054
-    AegisName: Assassin_Mask
-    Name: Assassin Mask
+    AegisName: Popcorn_Hair
+    Name: Popcorn Hair    # !todo check english name
     Type: Armor
     Buy: 20
-    Weight: 100
-    Jobs:
-      Assassin: true
-      Priest: true
+    Weight: 300
     Locations:
-      Head_Low: true
+      Head_Top: true
     ArmorLevel: 1
-    EquipLevelMin: 70
-    View: 180
+    View: 1895
```

Comment:
For me, the item shouldn't be changed in pre-re. Whoever wants to use pre-re should also use a pre-re client because there is a lot of change between the items description in pre-re and re

Reply:
but item ID 5054 affects both **custom/itemmail.txt** and **quests/seals/sleipnir_seal.txt** as both are used in PRE/RE